functions/Test-DbaVirtualLogFile.ps1

```diff
@@ -91,35 +91,35 @@ Returns VLF counts for the db1 and db2 databases on sqlcluster.
 		{
 			#For each SQL Server in collection, connect and get SMO object
 			Write-Verbose "Connecting to $servername"
-			$server = Connect-SqlServer $servername -SqlCredential $SqlCredential
+			try {
+				$server = Connect-SqlServer $servername -SqlCredential $SqlCredential
+			}
+			catch
+			{
+				Write-Warning "Can't connect to $instance, skipping..."
```

Comment:
that will probably be rewritten once we know for sure what to do. For the meantime, the behaviour is the same of other commands.